extensions/amp-story/amp-story-analytics.md

```diff
@@ -2,6 +2,31 @@
 
 This document details triggers associated with AMP Stories. If you're looking for a guide to setting up analytics for your AMP pages, see [this blog post](https://blog.amp.dev/2019/08/28/analytics-for-your-amp-stories/)
 
+## Shopping triggers
+
+### story_shopping_plp_view
+
+Triggers whenever the template displays the Product Listing Page without an active product.
+Will not fire when the Product Listing Page is viewed within the context of a Product Details Page.
+The event label will be the active page since there is no product associated with the Product Listing Page.
+
+event action: story_shopping_plp_view
+event label: active page id
+
+### story_shopping_pdp_view
+
+Triggers whenever the template displays the Product Details Page.
+
+event action: story_shopping_pdp_view
+event label: active product id
+
+### story_shopping_buy_now_click
+
+Triggers when the “Buy now” cta button is clicked”.
```

Comment:
Please remove the `"` at the end of the sentence.

We need to be specific about where the button is. 
`Triggers when the “Buy now” cta button in the Product Description Page is clicked.`